scripts/create-clog.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+#
+# create-clog.sh <version> <release-date>
+#
+# Dependencies: sed
+# 
+# Description: Uses template_changelog.md to create a new changelog.
+#
+# Examples:
+#  ./scripts/create-clog.sh "1.18.0" "05/18/2021"
+
+
+set -eou pipefail
+
+VERSION_DELIM="<% X.Y.Z %>"
+RELEASE_DATE_DELIM="<% MM/DD/YYYY %>"
+VERSION=""
+RELEASE_DATE=""
+
+# usage prints information for how to use this program
+usage () {
+  echo "Usage: create-clog <version> <release-date>"
+  echo "Create a changelog from a template"
+  echo "Arugments:"
```

Comment:
```suggestion
  echo "Arguments:"
```

Not sure if you meant Arugula :wink:

Reply:
I just ate some :green_salad: (albeit in smoothie form)

Reply:
also totally unrelated but I started writing unit tests for our bash scripts using BATS in the monorepo, lol